examples/typescript-backbone/js/app.ts

```diff
@@ -261,6 +285,28 @@ class TodoView extends Backbone.View {
 
 }
 
+// Todo Router
+// -----------
+
+class TodoRouter extends Backbone.Router {
+
+	routes = {
+		'*filter': 'setFilter'
+	};
+
+	constructor () {
+		super();
+		(<any>this)._bindRoutes();
+	}
+
+	setFilter (param: string) {
```

Comment:
Sorry, I forgot to push this. I was on an international flight and didn't have internet at the time. Should be good now.